oxidation/libparsec/crates/client_high_level_api/Cargo.toml

```diff
@@ -14,6 +14,7 @@ libparsec_crypto = { path = "../crypto" }
 libparsec_platform_async = { path = "../platform_async" }
 libparsec_platform_device_loader = { path = "../platform_device_loader" }
 libparsec_protocol = { path = "../protocol" }
+libparsec_testbed = { path = "../testbed" }
```

Comment:
`libparsec_testbed` should not be present in release, so we shouldn't have this unconditional add here !

Reply:
(see what is done for e.g. `libparsec_platform_device_loader`)